src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number,
```

Comment:
Consider adding JSDoc comments to describe the purpose of the interface and its properties. 📝